drivers_windows.go

```diff
@@ -16,6 +16,8 @@ func getInitializers(experimental bool) []initializer {
 		{windows.GetInit("l2bridge"), "l2bridge"},
 		{windows.GetInit("l2tunnel"), "l2tunnel"},
 		{windows.GetInit("nat"), "nat"},
+		{windows.GetInit("internal"), "internal"},
+		{windows.GetInit("private"), "naprivatet"},
```

Comment:
is this supposed to be private?

Reply:
This worked:
PS C:\Program Files\Debugging Tools for Windows (x64)> docker network ls
NETWORK ID          NAME                DRIVER              SCOPE
be554c4fbc1a        nat                 nat                 local
37a70ad722e5        none                null                local
778c644db3f6        test1               private             local

but I have fixed the typo